### PR TITLE
WIP: PLpeople popups

### LINKED ISSUE #421: Format marker popups for PLpeople
In LBLD I added a new functionality to have custom code for marker popups: https://github.com/publiclab/leaflet-blurred-location-display/pull/96

Here in LEL I want to use that functionality to customize the marker popups for PLpeople, to look more in line with the content popups in plots2.

### PR DESCRIPTION
**Pull request by nstjean**
Fixes #421  (<=== Add issue number here)

**This depends entirely on https://github.com/publiclab/leaflet-blurred-location-display/pull/96 and cannot be merged until that PR is merged and published in LBLD.**

The "Joined [time ago]" depends on new data added to the API in plots2: https://github.com/publiclab/plots2/pull/7562

This is what the new popups look like:

FireShot Capture 321 - Leaflet Environmental Layers -

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by nstjean** [comment]
FireShot Capture 325 - Leaflet Environmental Layers -

**Comment by jywarren** [comment]
OK! I've merged LBLD upstream and republished as `v1.3.0`, and have it working so you can click on the shaded boxes and that shows a list of people too!

https://github.com/publiclab/leaflet-blurred-location-display/pull/102 and https://github.com/publiclab/leaflet-blurred-location-display/pull/101

If we resolve conflicts here, are we ready to merge this? Thanks! Hope you're well!

File: src/util/timeago.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+TimeAgo = function TimeAgo() {
```
Comment (jywarren) [inline src/util/timeago.js:1]: This is really cool. Did you write this? You should consider releasing it as a stand-alone NPM module. It'd be useful and being maintainer on a small NPM module is a pretty interesting experience.

Comment (nstjean) [reply src/util/timeago.js:1]: @jywarren I didn't, it is from here: https://gist.github.com/caiotarifa/30ae974f2293c761f3139dd194abd9e5 but I had to modify it for what we needed!
I also used the modified version here: https://github.com/publiclab/plots2/blob/668c28e5822f17438b7b4da8d39e27d3af7f38a1/app/assets/javascripts/timeago.js

Comment (jywarren) [reply src/util/timeago.js:1]: Awesome. Would you have any time to try rebasing this to get it merged? I don't think it should be too tough... but i understand if you don't have time right now! Be safe and well!

**Comment by nstjean** [comment]
@jywarren NPM is showing blurred-location-display still at 1.1.0, can you publish the newest version? https://www.npmjs.com/package/leaflet.blurred-location-display

**Comment by jywarren** [comment]
Ah yes, sorry!

**Comment by jywarren** [comment]
Hi @sagarpreet-chadha -- actually i'm not a collaborator on LBLD - can you add me? Thank you!

**Comment by sagarpreet-chadha** [comment]
Hey Jeff, added you as maintainer :+1:

**Comment by jywarren** [comment]
OK! `v1.3.0` published!!! https://www.npmjs.com/package/leaflet.blurred-location-display

Thank you @sagarpreet-chadha !!!

**Comment by nstjean** [comment]
Great thanks!! I'll get this done.

File: src/util/timeago.js (hunk 1 above)
Comment (jywarren) [reply src/util/timeago.js:1]: Noting that TimeAgo is not appearing on publiclab.org, i think we need to add a JS include there somewhere... https://publiclab.org/map#11/40.67759191045513/-73.87728530215101

https://github.com/publiclab/plots2/pull/7715/files#diff-a9c3bd311eab80c9ebe6a69830f9ad02R62 seems to include it though... hmm

Comment (jywarren) [reply src/util/timeago.js:1]: Sorry to clog up this thread. I think it's a load-order issue and i'll re-order it. All good!

**Comment by jywarren** [comment]
OK, I believe I've gotten it working over in https://github.com/publiclab/leaflet-environmental-layers/pull/453 now after rebasing - whew, that was really a tough one! Fingers X that it works and if you can take a look over it @nstjean that would be awesome, but no worries!

**Comment by jywarren** [comment]
Solved in #453!

**Pull request closed without merging**